### PR TITLE
Wait for health

### PR DESCRIPTION
**Pull request by informalict**
When environment is available, it does not mean that healthiness can be fetched right away. Sometimes we must wait for it.

### CODE REVIEW

File: test/server_logs_test.go [hunk 1]
```diff
@@ -153,3 +154,44 @@ func changeLogLevel(l string) string {
 
 	return "INFO"
 }
+
+// waitForHealthT returns cluster health when environment is healthy.
+func waitForHealthT(t *testing.T, ctx context.Context, client driver.Cluster) driver.ClusterHealth {
+	ctxInner := ctx
+	if _, ok := ctx.Deadline(); !ok {
+		// When caller does not pride timeout, wait for healthiness for 30 seconds.
```
Comment (jwierzbo) [inline test/server_logs_test.go:162]:
```suggestion
		// When caller does not provide timeout, wait for healthiness for 30 seconds.
```

File: test/server_logs_test.go [hunk 2]
```diff
@@ -153,3 +154,44 @@ func changeLogLevel(l string) string {
 
 	return "INFO"
 }
+
+// waitForHealthT returns cluster health when environment is healthy.
```
Comment (jwierzbo) [inline test/server_logs_test.go:158]: we have already similar method: https://github.com/arangodb/go-driver/blob/0af2b60ac232c1a0a25abd67799394a9233f4919/test/client_test.go#L371-L371

Could you consider merging them?

File: v2/tests/admin_log_test.go [hunk 3]
```diff
@@ -76,65 +77,65 @@ func Test_LogLevelsForServers(t *testing.T) {
 	}
 
 	Wrap(t, func(t *testing.T, client arangodb.Client) {
-		withContext(time.Minute, func(ctx context.Context) error {
+		err := withContext(time.Minute, func(ctx context.Context) error {
```
Comment (jwierzbo) [inline v2/tests/admin_log_test.go:80]: please use `withContextT`

**Review by jwierzbo — CHANGES_REQUESTED** [review]

File: test/server_logs_test.go (hunk 1 above)
Comment (informalict) [reply test/server_logs_test.go]: changed

File: test/server_logs_test.go (hunk 2 above)
Comment (informalict) [reply test/server_logs_test.go]: changed

File: v2/tests/admin_log_test.go (hunk 3 above)
Comment (informalict) [reply v2/tests/admin_log_test.go]: changed

**Pull request merged**